### PR TITLE
Create Terratest to VPC module

### PR DESCRIPTION
**Pull request by amandasoaares**
* Create terratest to a VPC module
* Create simple-vpc test
* pre-commit
* Jenkinsfile
* MakeFile

### CODE REVIEW

File: examples/simple-vpc/outputs.tf [hunk 1]
```diff
@@ -10,11 +10,6 @@ output "vpc_cidr_block" {
   value       = module.vpc.vpc_cidr_block
 }
 
-//output "vpc_ipv6_cidr_block" {
-//  description = "The IPv6 CIDR block"
-//  value       = ["${module.vpc.vpc_ipv6_cidr_block}"]
-//}
-
```
Comment (danielcrisap) [inline examples/simple-vpc/outputs.tf:17]: 💅 
Delete instead comment 🙃

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        stage('Terraform Formatted') {
+            steps {
+                sh 'if [[ -n "$(terraform fmt -write=false)" ]]; then echo "Some terraform files need be formatted, run \'terraform fmt\' to fix"; exit 1; fi'
+            }
+        }
+        stage('Terraform Tflint') {
+            steps {
+                sh 'tflint'
+            }
+        }
```
Comment (danielcrisap) [inline Jenkinsfile:46]: I think it will be cleaner if we use `Makefile` commands, what do you think ?

File: test/simple_vpc_test.go [hunk 3]
```diff
@@ -1,26 +1,35 @@
...
-	}
 
+		Vars: map[string]interface{}{
+			"vpc_name": expectedName,
+			"region":   awsRegion,
+		},
+	}
 	// At the end of the test, run `terraform destroy` to clean up any resources that were created
-	defer terraform.Destroy(t, terraformOptions)
+	// defer terraform.Destroy(t, terraformOptions)
```
Comment (danielcrisap) [inline test/simple_vpc_test.go:32]: Is this right ? It will not destroy the test after apply.

File: Jenkinsfile (hunk 2 above)
Comment (amandasoaares) [reply Jenkinsfile]: yes

File: test/simple_vpc_test.go (hunk 3 above)
Comment (amandasoaares) [reply test/simple_vpc_test.go:32]: terraform destroy in module vpc gets error with localstack

I created an issue https://github.com/localstack/localstack/issues/2340

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -1,10 +1,11 @@
 repos:
   - repo: git://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.27.0
+    rev: v1.29.0
     hooks:
       - id: terraform_fmt
       - id: terraform_docs
```
Comment (wmaramos) [inline .pre-commit-config.yaml:6]: I think you can use `terraform_tflint` too

File: Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    options {
+      ansiColor('xterm')
+    }
+
+    stages {
+        stage('Install some packages') {
+            steps {
+                sh label: 'Install TfLint', returnStatus: true, script: '''
+                  if ! [ -x "$(command -v tflint)" ]; then
```
Comment (wmaramos) [inline Jenkinsfile:16]: Maybe you can create a new target in Makefile to install all necessary packages

**Pull request merged**